src/parse/converters/expressions/primary/literal/readRegexpLiteral.js

```diff
@@ -0,0 +1,17 @@
+import { REGEXP_LITERAL } from 'config/types';
+
+// as far as I can tell, this only falls over if modifiers are repeated erroneously
+var regexpPattern = /^\/(?:(?=\\\/)..|[^\/])+\/[gimy]{0,4}/;
```

Comment:
`/a\\/b/` and `/a[b/c]d/` will both fail. I don't think you can write a regexp for a regexp, but maybe I'm wrong.